forge-gui/res/cardsfolder/upcoming/resolute_veggiesaur.txt

```diff
@@ -0,0 +1,10 @@
+Name:Resolute Veggiesaur
+ManaCost:2 G
+Types:Creature Dinosaur Employee
+PT:3/3
+K:Trample
+T:Mode$ RolledDie | ValidPlayer$ You | TriggerZones$ Battlefield | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ3 | Execute$ TrigPutCounter | TriggerDescription$ Whenever you roll your third die each turn, put a +1/+1 counter on CARDNAME.
```

Comment:
Conditions aren't for triggers

Reply:
Tried it like that but didn't work. Probably need engine changes

Reply:
No, doesn't need any